Frontend/ADS_UI/src/app/change-time-modal/change-time-modal.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ModalController } from '@ionic/angular';
+import { EditOrder } from '../models/editOrder';
+import { OrderDetailed } from '../models/orderDetailed';
+import { OrderService } from '../services/order.service';
+
+@Component({
+  selector: 'app-change-time-modal',
+  templateUrl: './change-time-modal.component.html',
+  styleUrls: ['./change-time-modal.component.scss'],
+})
+export class ChangeTimeModalComponent implements OnInit {
+  editableDate = new EditOrder();
+  orderDetailed = new OrderDetailed();
+
+  // eslint-disable-next-line @typescript-eslint/member-ordering
+  @Input() id: string;
+  // eslint-disable-next-line @typescript-eslint/member-ordering
+  @Input() estimatedDate: string;
+
+  constructor(private modalController: ModalController,
+    private orderService: OrderService) { }
+
+  ngOnInit() {
+    console.log(this.estimatedDate);
+    // this.estimatedDate.setHours(0);
+    // this.estimatedDate.setMinutes(0);
+    // this.estimatedDate.setSeconds(0);
+    // this.estimatedDate.setMilliseconds(0);
```

Comment:
Console.log and comments left, i assume, from debugging (comments in some other places too)